docs/quickstart/quickstart_chains/ethereum-uniswap.md

```diff
@@ -0,0 +1,1111 @@
+# Ethereum Quick Start - Uniswap (Medium)
+
+## Goals
+
+Within the domain of DeFi, the objective of indexing Uniswap entities is both crucial and versatile. Essentially, indexing serves as the foundation for transparency and ease of use in the Uniswap system. By systematically organizing tokens, liquidity pools, transactions, and other essential information, indexing provides users with a quick and efficient means to search, find, and analyze data.
+
+The objective of this article is to offer a detailed, step-by-step guide on setting up a SubQuery indexer for Uniswap data indexing. We will comprehensively cover the necessary configurations and delve into the intricacies of the underlying logic.
+
+Previously, in the [1. Create a New Project](../quickstart.md) section, you must have noted [3 key files](../quickstart.md#_3-make-changes-to-your-project). Let's begin updating them one by one.
+
+## Setting Up the Indexer
+
+In this Uniswap indexing project, our main focus is on configuring the indexer to exclusively capture logs generated by three specific types of Uniswap smart contracts:
+
+1. **UniswapV3Factory** (contract address: `0x1F98431c8aD98523631AE4a59f267346ea31F984`): This contract is responsible for creating all the pool smart contracts.
+
+2. **Smart Contracts for Individual Pools**: These contracts represent individual liquidity pools.
+
+3. **NonfungiblePositionManager** (contract address: `0xC36442b4a4522E871399CD717aBDD847Ab11FE88`): This contract is instrumental in producing liquidity positions, which are implemented as NFTs. This functionality enables additional use-cases, including the transfer of liquidity positions.
+
+To gain a deeper understanding of how these core mechanisms work, you can refer to the official [Uniswap documentation](https://docs.uniswap.org/contracts/v3/reference/deployments).
+
+::: warning
+We suggest starting with the [Ethereum Gravatar example](./ethereum-gravatar). The ENS project is a lot more complicated and introduces some more advanced concepts
```

Comment:
Do you mean - The Ethereum Uniswap project is more complicated?  @bgodlin

Reply:
> Do you mean - The Ethereum Uniswap project is more complicated? @bgodlin

fixed as part of https://github.com/subquery/documentation/pull/415/commits/4d14525dd66b25bcc2e72c07f85e0c13969b1106